service-runner/src/main/resources/config/stellar.host.docker.internal.toml

```diff
@@ -0,0 +1,33 @@
+ACCOUNTS = ["GCSGSR6KQQ5BP2FXVPWRL6SWPUSFWLVONLIBJZUKTVQB5FYJFVL6XOXE"]
```

Comment:
Question: Why do we need a new config?

Reply:
`localhost` is not accessible. The TOML file must direct the stellar-anchor-tests in docker container to `host.docker.internal`.